libcaf_net/caf/net/transport_worker_dispatcher.hpp

```diff
@@ -140,13 +143,36 @@ class transport_worker_dispatcher {
   }
 
 private:
+  worker_ptr find_by_node(const node_id& nid) {
```

Comment:
```suggestion
  worker_ptr find_worker(const node_id& nid) {
```

Find by node... what? The signature makes it clear what our key is, so the function name should reflect what the function is actually doing.